app/src/main/java/com/example/lastfm/MainActivity.java

```diff
@@ -72,59 +71,44 @@ protected String doInBackground(String... strings) {
 
                 JSONObject topLevel = new JSONObject(builder.toString());
                 JSONObject main = topLevel.getJSONObject("artists");
-                System.out.println(main);
                 JSONArray artist = (JSONArray) main.get("artist");
-                System.out.println(artist);
+                if (artist != null) {
+                    for (int i=0;i<artist.length();i++){
+                        listofartists.add(artist.getString(i));
+                        System.out.println(artist.getString(i));
```

Comment:
read the link about logging in Android at last

Reply:
No more logging with println